blocks.ini

```diff
@@ -0,0 +1 @@
+woocommerce_blocks_phase = experimental
```

Comment:
I think this file should be added to `.gitignore` and not included in repo. It's more of a build artifact. This of course _does_ mean you need to address my comment with the `Bootstrap.php` changes for validating the value from `blocks.ini` before using it.